### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by lev-at**
- Updated EyesUtilities Dep.
- Fixed lf flag message being printed even when flag not set

### CODE REVIEW

File: src/main/java/com/applitools/imagetester/lib/EyesFactory.java [hunk 1]
```diff
@@ -15,211 +15,207 @@
...
+					? Utils.parseEnum(AccessibilityGuidelinesVersion.class, settings[1], "_")
+					: version;
+		}
+		this.accSettings = new AccessibilitySettings(level, version);
+		return this;
+	}
+
+	public EyesFactory logHandler(boolean logHandler) {
+		this.logHandler = logHandler;
+		return this;
```
Comment (valandi) [inline src/main/java/com/applitools/imagetester/lib/EyesFactory.java:214]: The contents of EyesFactory got tabbed over by one. Please fix that spacing, and this LGTM!

**Review by valandi — CHANGES_REQUESTED** [review]
Fix up the spacing, and I'll merge this in. 

Thanks!

File: src/main/java/com/applitools/imagetester/lib/EyesFactory.java (hunk 1 above)
Comment (lev-at) [reply src/main/java/com/applitools/imagetester/lib/EyesFactory.java]: How does it look now?

File: src/main/java/com/applitools/imagetester/lib/EyesFactory.java [hunk 2]
```diff
@@ -15,211 +15,207 @@
...
+					? Utils.parseEnum(AccessibilityGuidelinesVersion.class, settings[1], "_")
+							: version;
+		}
+		this.accSettings = new AccessibilitySettings(level, version);
+		return this;
+	}
+
+	public EyesFactory logHandler(boolean logHandler) {
+		this.logHandler = logHandler;
+		return this;
```
Comment (valandi) [inline src/main/java/com/applitools/imagetester/lib/EyesFactory.java:214]: This is all still tabbed over. 

Can you replace with spaces?

**Review by valandi — CHANGES_REQUESTED** [review]
The EyesFactory class is still tabbed over.
Can you replace with spaces so that only the code that was changed is diffed?

File: src/main/java/com/applitools/imagetester/lib/EyesFactory.java (hunk 2 above)
Comment (lev-at) [reply src/main/java/com/applitools/imagetester/lib/EyesFactory.java]: Done.

**Pull request merged**